docs/HACKING.md

```diff
@@ -300,3 +300,30 @@ It will fail if you use e.g. `:latest`.
 
 At the time of this writing, the [kubelet](https://github.com/smarterclayton/origin/blob/4de957b019aee56931b1a29af148cf64865a969b/images/os/Dockerfile)
 has code to do this in CI, and work is in progress to [replicate that for cri-o](https://github.com/openshift/release/pull/4030).
+
+### Running worker nodes with customized kubelet
+
+1) Ssh to the node (use github.com/eparis/ssh-bastion)
+``
+$ ssh core@<worker-node-ip>
+``
+2) Use scp to copy it the customised kubelet binary
+``
+$ scp root@<remote-host-with-kubelet-binary>:<path/to/kubelet/binary> /home/core
+``
+3) Make the host writable
+``
+$ mount -o remount,rw /usr
```

Comment:
It's better to use `rpm-ostree usroverlay` as is mentioned above.

Reply:
Sure, I can update this to use `rpm-ostree usroverlay`